lib/output.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+let cli = require('heroku-cli-util')
+
+let Stream = function (url) {
```

Comment:
I've actually gone farther and removed that extraction. That plugin will probably never have more commands, so there's no need to add too many extractions.